tests/test_django_helpers.py

```diff
@@ -0,0 +1,122 @@
+"""
+Test django_reporting_helpers functions that aren't exercised elsewhere.
+"""
+import sys
+
+import mock
+
+from code_annotations.django_reporting_helpers import (
+    get_models_requiring_annotations,
+    is_non_local,
+    requires_annotations,
+    setup_django
+)
+from tests.fake_models import (
+    FakeBaseModelAbstract,
+    FakeBaseModelBoring,
+    FakeBaseModelBoringWithAnnotations,
+    FakeBaseModelProxy
+)
+
+
+@mock.patch('code_annotations.django_reporting_helpers.issubclass')
+def test_requires_annotations_abstract(mock_issubclass):
+    """
+    Abstract classes should not require annotations
+    """
+    mock_issubclass.return_value = True
+    assert requires_annotations(FakeBaseModelAbstract) is False
+
+
+@mock.patch('code_annotations.django_reporting_helpers.issubclass')
+def test_requires_annotations_proxy(mock_issubclass):
+    """
+    Proxy classes should not require annotations
+    """
+    mock_issubclass.return_value = True
+    assert requires_annotations(FakeBaseModelProxy) is False
+
+
+@mock.patch('code_annotations.django_reporting_helpers.issubclass')
+def test_requires_annotations_normal(mock_issubclass):
+    """
+    Non-abstract, non-proxy models should require annotations
+    """
+    mock_issubclass.return_value = True
+    assert requires_annotations(FakeBaseModelBoring) is True
+
+
+def test_requires_annotations_not_a_model():
+    """
+    Things which are not models should not require annotations
+    """
+    assert requires_annotations(dict) is False
```

Comment:
Nit: Throw in a few other key types?

Reply:
I don't think that would get us anything. I think we're just checking the `else` case here.